azDeploySecureSub.bicep

```diff
@@ -36,6 +36,12 @@ param redcapCommunityUsername string
 @description('REDCap Community site password for downloading the REDCap zip file.')
 @secure()
 param redcapCommunityPassword string
+@description('Github Repo URL where build scripts are downloaded from')
+param scmRepoUrl string = 'https://github.com/microsoft/azure-redcap-paas'
+@description('Github Repo Branch where build scripts are downloaded from')
+param scmRepoBranch string = 'main'
+@description('The prerequsites command before build to be run on the web app with an elevated privilege. This is used to install the required packages for REDCap.')
+param preRequsitesCommand string = 'apt-get install unzip -y && apt-get install -y python3 python3-pip'
```

Comment:
This is the command where we might need to copy the ngnix site configuration from `/home` to `/etc` (not exact paths)

Reply:
would you be specific how we want to do that?

Reply:
There will be a customized file somewhere in `/home` that's going to contain the custom configuration needed for nginx to serve REDCap correctly. On container start, we'll need to copy that file and overwrite the default. Then we'll also need to reload nginx.

`cp /home/nginx-sites-enabled-default /etc/nginx/sites-enabled/default`

We need to discuss where the file will come from. It might become part of the repo, or it might be that on first run, we need to customize it.

See https://techcommunity.microsoft.com/t5/apps-on-azure-blog/configure-nginx-for-php-8-linux-azure-app-service/ba-p/3069373